config/config.go

```diff
@@ -65,6 +67,8 @@ type Security struct {
 	Constructor interface{}
 }
 
+type ICEServer = webrtc.ICEServer
```

Comment:
Nope. Just so that we redefine it a different way later if we want. But I think it's fine to remove this and just take the pion specific object. We can make a breaking change if this needs to be changed.

Reply:
It’s a breaking change anyway if we change the underlying type. That should be fine.

Reply:
Can you remove it?